packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -282,6 +288,14 @@ export class WaferMap<
         return this.diesTable !== undefined;
     }
 
+    public async setData(tableData: Table): Promise<void> {
```

Comment:
Talking to @atmgrifter00 we should make a tech debt to do data validation as part of `setData` after the `dies` and `diesTable` properties are removed. That way for validation related to data we can throw an error instead of have users needing to inspect the validity state (other validity based on properties would still be part of the validity state).

Reply:
updated the list here https://github.com/ni/nimble/issues/2034